### PR TITLE
Ifteda Resource Page Update

### PR DESCRIPTION
**Pull request by ifteda**
Changes
- linked email in last section
- adjusted spacing and responsiveness to match About page (centered for mobile - is that okay?)

Desktop:
screencapture-localhost-3000-resources-1594582753894

Mobile:
localhost_3000_resources (1)

### CODE REVIEW

**Comment by ifteda** [comment]
Just noticed that the last divider is a different size. Fixing that now

File: src/styling/ResourcesPage.css [hunk 1]
```diff
@@ -22,8 +22,26 @@ a:hover {
 .icon {
     width: 100%;
     max-width: 300px;
+    height: auto;
```
Comment (feliciazhang) [inline src/styling/ResourcesPage.css]: can you change the tabbing in this file to 2 spaces please?

File: src/styling/ResourcesPage.css [hunk 2]
```diff
@@ -22,8 +22,26 @@ a:hover {
...
 
 .padding-r {
     padding-right: 50px;
+}
+
+@media only screen and (max-width: 768px) {
+    .icon {
+        max-width: 250px;
+      }
+      .padding-r {
```
Comment (feliciazhang) [inline src/styling/ResourcesPage.css]: I think the tabbing/spacing in this section is a bit off in general

File: src/pages/AboutPage.js [hunk 3]
```diff
@@ -108,32 +108,36 @@ const AboutPage = () => (
...
-        {/* FIXME update */}
+        <p>Want to get involved, or have questions, thoughts, feedback, or concerns?</p>
+        Please use one of the following subject line patterns in your emails:
+        <br />
+        "Volunteer: [Question or Feedback] [Subject]"
+        <br />
+        "Project Request: [Question or Feedback] [Subject]"
+        <br />
+        <br />
+        Email us at: {/* FIXME update */}
```
Comment (feliciazhang) [inline src/pages/AboutPage.js]: do we still need the fixme comment?

Comment (ifteda) [reply src/pages/AboutPage.js]: had removed them in the other PR. merging dev into this branch would probably fix it

**Review by liu-caro — APPROVED** [review]

**Pull request merged**